docs/tutorial/index.md

```diff
@@ -13,62 +13,36 @@ tags:
 search:
   boost: 2
 ---
-# Tutorial Overview
+# Tutorial: Master workflow automations
 
-This tutorial provides a guided walk-through of Prefect core concepts and instructions on how to use them.
+This tutorial helps you get started with Prefect and become familiar with developing, deploying, and managing Prefect workflows.
 
-By the end of this tutorial you will have:
+After you build your deployment, you can read other [guides](/guides) to learn how to integrate other services and add more features to your workflows.
```

Comment:
```suggestion
Once you've deployed your first workflow, you can read other [guides](/guides) to learn how to apply Prefect's features to more specific problems.
```

Reply:
The main change I'm advocating for here is that we want to use the word "deploy" as a verb rather than a noun.

Reply:
Maybe move the section on "After..." to the end of the tutorial, as that seems like a good time to direct folks to what they can do next.